### DISCUSSION TITLE
Using Custom Schemas while getting data through API

### DISCUSSION THREAD

**Discussion by srichint1** [supabase/supabase#20480] [Questions] [answered]
Hi
I have a custom schema called 'reporting' in my supabase project. It has couple of tables and views. I followed the steps defined in below link to allow fetching data through API (using anon role)
https://supabase.com/docs/guides/api/using-custom-schemas?language=curl
Even after this I am getting error where it is not recognizing the schema name.
Please suggest if I need to make any changes in my request
Thanks
![supabase_custom_schema_dataAPI_error](https://github.com/supabase/supabase/assets/120005578/3882fa43-583c-4f6d-9946-1caf4d16e85d)

**Comment by srichint1** [comment]
Sorry, Please see below error message (I pasted a wrong message earlier)
Thanks
![supabase_custom_schema_dataAPI_error_2](https://github.com/supabase/supabase/assets/120005578/75ad1a2b-a4a4-4576-9a4f-03f4610ca759)

**Comment by steve-chavez** [comment, marked as answer]
Accept-Profile is a special header for changing schemas, it's not a query string parameter. Remove it from the URL and you'll clear the error.

For more details, see https://postgrest.org/en/stable/references/api/schemas.html#multiple-schemas
